articles/user-guide/libraries/standard/diagnostics.md

```diff
@@ -88,12 +89,12 @@ That is,
 \begin{align}
     \ket{\psi} = \ket{\psi'} \text{ if and only if } \braket{\psi | \psi'} = 1.
 \end{align}
-Using the primitive operations defined in the prelude, we can directly perform a measurement that returns `Zero` if $\ket{\psi}$ is an eigenstate of one of the Pauli operators.
+Using the primitive operations defined in the prelude, you can directly perform a measurement that returns `Zero` if $\ket{\psi}$ is an eigenstate of one of the Pauli operators.
 
 
-The operation <xref:microsoft.quantum.diagnostics.assertqubit> provides a particularly useful shorthand to do so in the case that we wish to test the assertion $\ket{\psi} = \ket{0}$.
-This is common, for instance, when we have uncomputed to return ancilla qubits to $\ket{0}$ before releasing them.
-Asserting against $\ket{0}$ is also useful when we wish to assert that two state preparation `P` and `Q` operations both prepare the same state, and when `Q` supports `Adjoint`.
+The operation <xref:microsoft.quantum.diagnostics.assertqubit> provides a particularly useful shorthand to do so in the case that you want to test the assertion $\ket{\psi} = \ket{0}$.
+This is common, for example, when you have uncomputed to return ancilla qubits to $\ket{0}$ before releasing them.
```

Comment:
Somehow this use of "ancilla" escaped earlier edits; can we modify to "auxiliary?"